test/injection/core/registerTwiceError.test.js

```diff
@@ -0,0 +1,18 @@
+import { $injector } from './testsConfig';
+var mockService =  { get: 'I\'m a mock. ' };
```

Comment:
🧐 we don't want vars anymore 😉

Reply:
ah yes we make `let`/`const` not `var`... indeed :)